app/Http/Resources/ItemResource.php

```diff
@@ -81,15 +81,24 @@ private function getDescription()
 
     private function getAuthor()
     {
-        if ($this['item']->author_name) {
-            return $this['item']->author_name;
-        }
+        $localAuthoritiesNames = explode(', ', $this['item']->author_name);
+        $webumeniaAuthoritiesNames = collect($this['webumenia_item']->authorities)->map(
+            fn(object $authority) => formatName($authority->name)
+        );
+        $filteredLocalAuthoritiesNames = collect($localAuthoritiesNames)->filter(
+            fn($name) => !$webumeniaAuthoritiesNames->contains($name)
+        );
```

Comment:
```suggestion
        $webumeniaAuthorities = collect($this['webumenia_item']->authorities);
    
        $filteredLocalAuthoritiesNames = collect($localAuthoritiesNames)
            ->reject(fn ($name) => $webumeniaAuthorities->contains(formatName($name)));
```

`reject` is more expressive in terms of intent.

Reply:
This will not work as $webumeniaAuthorities is a collection of objects (or something) and other is collection of strings